main/core.js

```diff
@@ -42,16 +47,59 @@ async function setup (/** @type {Context} */ ctx) {
     }
   }
   await maybeMigrateFiles()
-  await start(ctx)
+  subscribeWithRetry(ctx).catch(console.error)
+  await start()
 }
 
-async function start (/** @type {Context} */ ctx) {
+async function subscribeWithRetry (/** @type {Context} */ ctx) {
```

Comment:
I think it would be better to implement this retry mechanism inside `core` APIs, as the `core` knows better how to detect whether the main process is already running or whether we need to wait until it starts.

Just an idea to consider.

Reply:
Core actually doesn't know if the process is running, and doesn't care if it does: It can still read historic events. These methods just tail files, so I don't think `core` needs to retry

Reply:
I see. So why do we have to retry here, in Station Desktop?

If somebody else decides to use the API provided by `@station/core` to integrate Station Core into their system, will they need to implement the same retry logic as we do in Station Desktop? If yes, then I think the retry logic belongs to Core. If not, then I am even more curious why Desktop needs this retry mechanism. 🙃 

Anyhow, this is not blocking this PR from being merged.

Reply:
> So why do we have to retry here, in Station Desktop?

We don't _have_ to, it's just so that we can recover in case any error is thrown there. Would you rather make it crash hard?

It's like with any stream or iterator: Consuming it _can_ fail, and it's up to the consumer to decide what to do about it